src/views/darkroom.c

```diff
@@ -436,7 +435,7 @@ void expose(
     else
     {
       fontsize = DT_PIXEL_APPLY_DPI(14);
-      load_txt = dt_util_dstrcat(NULL, "%s `%s' ...", _("loading"), dev->image_storage.filename);
+      load_txt = dt_util_dstrcat(NULL, NC_("darkroom", "%s `%s' ..."), _("loading"), dev->image_storage.filename);
```

Comment:
similarly here the string could be just "loading `%s' ..."

Reply:
Agreed too !